misk-feature-testing/src/main/kotlin/misk/feature/testing/TestFeatureFlagsModule.kt

```diff
@@ -0,0 +1,41 @@
+package misk.feature.testing
+
+import misk.feature.FeatureFlags
+import misk.feature.FeatureService
+import misk.ServiceModule
+import misk.inject.KAbstractModule
+import misk.inject.toKey
+import kotlin.reflect.KClass
+
+/**
+ * Binds a [TestFeatureFlags] that allows tests to override values.
+ */
+class TestFeatureFlagsModule(
+  private val qualifier: KClass<out Annotation>? = null
+) : KAbstractModule() {
+  private val testFeatureFlags = TestFeatureFlags()
```

Comment:
This class has an injectable constructor, you shouldn’t instantiate it manually

Reply:
I instantiate manually to enable the `withOverrides` thing.